Remove deprecated app documentation (#9228)
PostHog/posthog.com: contents/docs/advanced/cdp.md

> If you need destinations that we don't currently support, you have several options

Reviewer: ```suggestion
If you need destinations that we don't currently support, you have two options:
```

> 1. Build your own in-app destination [real-time destinations](/docs/cdp/destinations) by writing some custom logic to call the 3rd party API.

Reviewer: ```suggestion
1. Build your own [real-time destinations](/docs/cdp/destinations) to call the 3rd party API.
```

> 2. You can send your PostHog data [to a 3rd party CDP](#sending-posthog-data-to-a-3rd-party-cdp-for-extra-destinations).
> 2. Fully transition to a 3rd party CDP.

Reviewer: The only way to do this will be to use real time exports so might as well just be one option
```suggestion
2. Fully transition to a 3rd party CDP.
```

If there's destination you are looking for that we don't currently have, you have two options:

> 1. Build your own [real-time destinations](/docs/cdp/destinations) using custom code. You can also request a new destination in the app and our team will be notified.

Reviewer: ```suggestion
1. Build your own [real-time destinations](/docs/cdp/destinations) using custom code. You can also request a new destination by clicking **Can't find what your looking for?** when [creating a new destination](https://us.posthog.com/pipeline/new/destination) in-app and our team will be notified.
```
